tutorial/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ networks:
 services:
 
   alarms-service:
-    image: iotechsys/alarms-service:0.1
+    image: iotechsys/alarms-service:0.2 
```

Comment:
why hardcode one and env var the other?

Reply:
The environment variable does work with the edgecentral docker compose haha